src/applications/ivc-champva/10-7959C/components/ApplicantMedicareAdvantagePage.jsx

```diff
@@ -0,0 +1,53 @@
+import { pageProps, reviewPageProps, yesNoOptions } from '../config/constants';
+
+import ApplicantRelationshipPage, {
+  ApplicantRelationshipReviewPage,
+  appRelBoilerplate,
+} from '../../shared/components/applicantLists/ApplicantRelationshipPage';
+
+const KEYNAME = 'applicantMedicareAdvantage';
+const PRIMARY = 'hasAdvantage';
+const SECONDARY = '_unused';
+
+export function generateOptions({ data, pagePerItemIndex }) {
+  const bp = appRelBoilerplate({ data, pagePerItemIndex });
+  const prompt = `Did ${
+    bp.useFirstPerson ? 'you' : `${bp.applicant}`
+  } choose the advantage plan for coverage?`;
+
+  return {
+    ...bp,
+    options: yesNoOptions,
+    relativeBeingVerb: `${bp.relative} ${bp.beingVerbPresent}`,
+    customTitle: `${
+      bp.useFirstPerson ? `Your` : `${bp.applicant}’s`
+    } Medicare coverage`,
+    customHint:
+      'You can find this information on the front of your Medicare card.',
+    description: prompt,
+  };
+}
+
+export function ApplicantMedicareAdvantagePage(props) {
+  const newProps = {
+    ...props,
+    keyname: KEYNAME,
+    primary: PRIMARY,
+    secondary: SECONDARY,
+    genOp: generateOptions,
+  };
+  return ApplicantRelationshipPage(newProps);
+}
+export function ApplicantMedicareAdvantageReviewPage(props) {
+  const newProps = {
+    ...props,
+    keyname: KEYNAME,
+    primary: PRIMARY,
+    secondary: SECONDARY,
+    genOp: generateOptions,
+  };
+  return ApplicantRelationshipReviewPage(newProps);
+}
```

Comment:
Have to wrap the call to `ApplicantRelationshipPage` because we can't technically re-use the same custom page for multiple locations in the form. So here we adjust a couple props and return an instance of ApplicantRelationshipPage wrapped